tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -10,15 +10,17 @@ Feature: Change data of space
     Given user "Alice" has been created with default attributes and without skeleton files
     And the administrator has given "Alice" the role "Admin" using the settings api
 
-  Scenario: An admin user can change the name of a Space via the Graph API
+  Scenario: An admin user can change the name and description of a Space via the Graph API
```

Comment:
This scenario is good. But is it also possible to specify the description at the time that a space is created?
If so, then it might be worth adding a feature `createSpaces.feature` that demonstrates creating a space with all its parameters specified (Name, driveType, quota, description) in a single create-space API call.

Reply:
yes. let us add it later. Currently we have no step definition for that